src/data/courses.data.json

```diff
@@ -0,0 +1,123 @@
+[
```

Comment:
Is this fake data?
Do we need it anymore?

Reply:
Currently only the playlists and videos still need this.